### ISSUE TITLE
how to do a post call only once

### ISSUE THREAD

**Issue by vishalbiradar** [shoreditch-ops/artillery#418]
I want to execute the login post call only once and the next task post calls multiple times.
But with below code, the number of times I am calling for task post call those many times the login post call is getting executed.
But I want the login post call to be executed only once.
So below is my code:

```

config:
  target: "http://localhost:1337"
  phases:
    - duration: 1 
      arrivalRate: 1 
    - duration: 10
      arrivalRate: 10

scenarios:
- name: "Sign up a user and get the token"
  flow:
    - post:
        url: "/auth/login"
        json:
          email: "mymail@gmail.com"
          passwordHash: "password"
        capture:
          - json: "$.data.api_key"
            as: "api_key"
    - post:
        url: "/task"
        json:
          taskType: "CHECK_CONNECTION"
        headers:
          api_key: "{{ api_key }}"
        capture:
          - json: "$.data.id"
            as: "taskId"
    - log: "New Task Created with id= {{ taskId }}"
```

How to do it??
Please help me...

**Comment by hassy** [comment]
You can make that auth call once, save the API key into a CSV file and read the value in with `config.payloads` (see the docs). Artillery does not have a built in concept of a pre-test task.

**Comment by hassy** [comment]
You need to write a shell script to be able to grab auth details only once per Artillery run.

The rest is covered in the docs, please take the time to read them.

**Comment by vishalbiradar** [comment]
At least help me in the below scenario.

If I want to make three calls,
First Api call = only once
Second Api call = 10 times
Third Api call = 200 times

Then how the yml file will look like ??
